packages/desktop/src/lib.rs

```diff
@@ -19,6 +19,17 @@ mod webview;
 #[cfg(any(target_os = "ios", target_os = "android"))]
 mod mobile_shortcut;
 
+// WARN about wry support on windows gnu targets. GNU windows targets don't work well in wry currently
```

Comment:
Don't people usually stick this check in build.rs?

Reply:
That is true, build scripts let us create a much nicer looking warning message. I have been writing too many macros 😆